source/js/interview.js

```diff
@@ -185,9 +185,9 @@ $(document).ready(function () {
 
   // Function that's responsible for generating a hash from a string
   function hashCode(string) {
```

Comment:
there are many different types of hashes for different purposes, what are the properties / purposes of this hash function?

Reply:
to generate a hash code that can be used to identify certain things

Reply:
the sole purpose and function of this function is to just generate a hash from given string